gradio/exceptions.py

```diff
@@ -0,0 +1,4 @@
+class Error(Exception):
```

Comment:
I think it's good to add a `__str__()` method, following this tutorial: https://docs.python.org/3.3/tutorial/errors.html#user-defined-exceptions